CMake/PackageRedhat.cmake

```diff
@@ -0,0 +1,34 @@
+if (UNIX)
+    set(CPACK_GENERATOR "RPM")
+    set(CPACK_SOURCE_TGZ "ON")
+
+    set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "FlatBuffers serialization library and schema compiler.")
+    
+    set(CPACK_RPM_PACKAGE_HOMEPAGE "https://github.com/google/flatbuffers")
+    set(CPACK_RPM_PACKAGE_MAINTAINER "Vitaly Isaev <vitalyisaev2@gmail.com>")
```

Comment:
Is this person really the maintained for the Redhat package?

Reply:
This was cribbed from the DEB package `.cmake`. Is there a suitable catchall email? I'm unclear how to refer to the Google Group in this entry.

Reply:
1. It worth noting that `flatbuffers` package already exists on [Fedora Koji](https://koji.fedoraproject.org/koji/packageinfo?packageID=25535), so you can either take this package, or take `src.rpm` and rebuild it for any RHEL-based distribution you need. The only meaningful reason to add CPack-based RPM package here is that if you use RPM packages in your CI and you need to have flatbuffer's `master`. In case if you need stable version, I'd prefer to use packages provided by Fedora team.
2. I believe you should put your name and email here instead of mine :)